tests/scripts/create-dev-cluster.sh

```diff
@@ -163,13 +172,10 @@ while getopts ":hrmfd:p:" opt; do
 	    minikube_profile_name="$OPTARG"
 	    ;;
 	\?)
-	    echo  "Invalid option: -$OPTARG" >&2
-	    show_usage
-	    exit 1
+	    invocation_error " Invalid option: -$OPTARG"
```

Comment:
extra space before "Invalid"

Reply:
@rkachach  wrote:
> extra space before "Invalid"

if I do that, the line reads: 
```
   invocation_error"Invalid option: -$OPTARG:
```
 and I obviously get an error trying to execute it triggering this code path:

```
$ tests/scripts/create-dev-cluster.sh -x
tests/scripts/create-dev-cluster.sh: line 173: invocation_error Invalid option: -x: command not found
$
```

Reply:
I the mean the space in " Invalid..." (at the the begging)